Sources/UInt128.swift

```diff
@@ -37,43 +37,35 @@ public enum UInt128Errors: Error {
     /// 128 bits.
     case stringInputOverflow
 }
+
 // MARK: Data Type
 /// A 128-bit unsigned integer value type.
 /// Storage is based upon a tuple of 2, 64-bit unsigned integers.
 public struct UInt128 {
     // MARK: Type Properties
     /// The largest value a UInt128 can hold.
-    public static var max: UInt128 {
-        return UInt128(upperBits: UInt64.max, lowerBits: UInt64.max)
-    }
+    public static let max: UInt128 = UInt128(hi: .max, lo: .max)
```

Comment:
As an FYI; I did some research and found out that since they are static, they are lazily loaded by default, even if they are not computed values, so, my initial worries on this were unfounded.